db/resources/bin/pg_backup.sh

```diff
@@ -96,7 +96,13 @@ function perform_backups()
 				mv $FINAL_BACKUP_DIR"$DATABASE".sql.gz.in_progress $FINAL_BACKUP_DIR"$DATABASE".sql.gz
 			fi
 			set +o pipefail
-                        
+
+			if (( $(stat -c%s $FINAL_BACKUP_DIR"$DATABASE".sql.gz) > 10000000 )); then
```

Comment:
Minor: consider replacing the byte number by a variable `BYTE_COUNT` - otherwise this is a "magic number"

Reply:
Very good point. 

Please see latest commit.
Added the variable and short comment.

Also moved the file size check behind the plain and custom backup blocks.
Removed duplicate info echo.